src/backend/commands/taskcmds.c

```diff
@@ -109,6 +109,14 @@ DefineTask(ParseState *pstate, CreateTaskStmt *stmt)
         }
     }
 
+	if (!allowSystemTableMods && strncmp(stmt->taskname, DYNAMIC_TASK_PREFIX, 25) == 0)
```

Comment:
align code.

Reply:
This is a right align with tab of Postgres, however the original codes ahead/behind have align problem.
You could find it in a IDE like VScode or vim. 
The pg_task codes may need a pgindent at a right time.